swagger.yaml

```diff
@@ -489,6 +494,9 @@ definitions:
         enum:
           - dataset-api
           - zebedee
+      Etag:
```

Comment:
This should not exist in the API response data, the ETag is only returned as a header - please remove it thanks